docs/src/meshes/mesh_data_meshes.md

```diff
@@ -123,3 +123,8 @@ Some key elixirs to look at:
 * `elixir_euler_triangulate_pkg_mesh.jl`: uses a `TriangulateIO` unstructured mesh generated by `Triangulate.jl`.
 * `elixir_ape_sbp_triangular_mesh.jl`: uses a multi-dimensional SBP discretization in weak form.
 * `elixir_euler_tet_mesh.jl`: basic weak form DG discretization on a uniform tet mesh.
+
+We also have support for flux differencing on simplicial meshes:
+* `elixir_euler_triangular_mesh_flux_diff.jl`: Modal DG discretization with flux differencing on a uniform triangular mesh.
+* `elixir_euler_sbp_triangular_mesh_flux_diff.jl`: SBP-DG discretization with flux differencing on a uniform triangular mesh.
+* `elixir_euler_tet_mesh_flux_diff.jl`: Modal DG discretization with flux differencing on a uniform tetrahedral mesh.
```

Comment:
Are these elixirs using entropy-conservative methods? If so, you could use the same naming scheme as we do for the other meshes (`ec` instead of `flux_diff`).

Reply:
For reference, is there a similar naming convention for entropy stable elixirs (e.g., with flux dissipation)?

Reply:
We sometimes use `_es` (entropy-stable) instead of `_ec` (entropy-conservative). However, most elixirs where we use entropy-dissipative methods are more like applications where something else is in the focus. For example, the point of the `_ec` elixirs is to check whether everything works by testing EC. Methods where we want to use dissipative schemes can be, for example, applications to blast waves or something similar.